test_scripts/Defects/8_0/3698_SDL_rejects_invalid_PTU.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3698
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL rejects invalid PTU received from App and doesn't crash
+--
+-- Steps:
+-- 1. New app is registered
+-- 2. SDL does start PTU sequence
+-- 3. App provides invalid PTU during update
+-- SDL does:
+--  - not crash
+--  - reject update
+--  - switch to 'UPDATE_NEEDED' state
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+local utils = require("user_modules/utils")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.ExitOnCrash = false
+
+--[[ Local Variables ]]
+local ptuContent = '"crash"'
+
+--[[ Local Functions ]]
+local function expMsgs()
+  common.hmi.getConnection():ExpectNotification("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED" })
```

Comment:
Maybe better use: common.hmi.getConnection() => common.getHMIConnection 
```suggestion
  common.getHMIConnection:ExpectNotification("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED" })
```

Reply:
@VNikiforov-dev I propose to avoid using old functions of actions module like actions.getHMIConnection() because it make harder future refactoring of scripts. Please use actions.hmi.getConnection instead of actions.getHMIConnection()

Reply:
@aderiabin I got it. Thanks for your help!